src/pages/library/common.js

```diff
@@ -354,7 +384,7 @@ export const SearchAndFilterComponent = (featuredList, sidebarSections, activeTa
         ])
       ]),
 
-    selectionActionComponent(selectedData, setSelectedData)
+    SelectionActionComponent(selectedData, setSelectedData)
```

Comment:
We use components with the following format:

```suggestion
    h(SelectionActionComponent, { selectedData, setSelectedData })
```